mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMessageIdManager.java

```diff
@@ -212,9 +219,12 @@ public void setInMailboxesNoCheck(MessageId messageId, MailboxId targetMailboxId
 
     private List<MailboxMessage> findRelatedMailboxMessages(MessageId messageId, MailboxSession mailboxSession) throws MailboxException {
         MessageIdMapper messageIdMapper = mailboxSessionMapperFactory.getMessageIdMapper(mailboxSession);
-        return messageIdMapper.find(ImmutableList.of(messageId), MessageMapper.FetchType.Full)
```

Comment:
It looks like, for this specific method, we don't gain anything because we handle a single message.
The cood is then more complicated then before without any benefits.
but I may missed something

Reply:
True. I will revert this specific change.